configs/qvhighlight/clip_slowfast_moment_detr_qvhighlight_pretrain.yml

```diff
@@ -0,0 +1,19 @@
+model_name: moment_detr
+dset_name: qvhighlight_pretrain
+ctx_mode: video_tef
+v_feat_types: slowfast_clip
+t_feat_type: clip
+train_path: data/qvhighlight/subs_train.jsonl
+eval_path: data/qvhighlight/highlight_val_release.jsonl
+eval_split_name: val
+v_feat_dirs: ['features/QVHighlight/clip', 'features/QVHighlight/slowfast']
+t_feat_dir: features/QVHighlight/clip_text_subs_train/moment_detr
```

Comment:
Could you explain about the directory structure of t_feat_dir? You specify it "features/QVHighlight/clip_text_subs_train/moment_detr", but "moment_detr" is necessary?

Reply:
I made additional "moment_detr" directory because i am going to introduce another ASR-based pre-training data, e.g., generated by whisper-x data. Therefore, it would not be necessary if not use another ASR-based pre-training data.

Reply:
Got it. So, could you rename "moment_detr" into "original_asr"?

Reply:
I modify the t_feat_dir path to "features/QVHighlight/clip_text_subs_train"